hack/addons/mindthegap-helm-registry/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ghcr.io/mesosphere/mindthegap:v1.13.4
+# this gets called by goreleaser so the copy source has to be the path relative to the repo root.
+RUN --mount=source=./hack/addons/mindthegap-helm-registry/repos.yaml,target=/repos.yaml ["/ko-app/mindthegap", "create", "helm-bundle", "--helm-charts-file=/repos.yaml", "--output-file=/tmp/helm-charts.tar"]
+VOLUME /certs
+CMD ["serve", "bundle", "--bundle=/tmp/helm-charts.tar", "--listen-port=5000", "--listen-address=0.0.0.0", "--tls-private-key-file=/certs/tls.key", "--tls-cert-file=/certs/tls.crt"]
```

Comment:
i guess. this is already very tightly coupled with the build process and release and deployments can always override this so i don't really see a good reason to remove it.

Reply:
> I think this would fit better in the Deployment since it ultimately makes the decision about where to mount the volume and the port to run on

+1

Reply:
either is fine because you can override entrypoints and commands in any container. this seems like a very small nit which has no implications on the product or extensibility of it.